reportssql/testing_report_functions.sql

```diff
@@ -269,6 +269,36 @@ WHERE
 END$$ 
 DELIMITER ;
 
+DROP FUNCTION IF EXISTS Testing_Indicator5j;
+
+DELIMITER $$
+CREATE FUNCTION Testing_Indicator5j(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAge INT(11),
+    p_endAge INT (11),
+    p_includeEndAge TINYINT(1),
+    p_gender VARCHAR(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+    DECLARE previousHIVTestDateFromCounselingForm DATE;
+
+    SELECT
+        COUNT(DISTINCT pat.patient_id), getPatientPreviousHIVTestDateFromCounselingForm(pat.patient_id)
+        INTO
+        result, previousHIVTestDateFromCounselingForm
```

Comment:
@lauravignoli we'll need a fix similar to https://github.com/Jembi/cameroon-bahmni-metadata/commit/955bb13f47f4b8444f4e0bc3020895c42d6ee109 here as well